code/modules/mob/living/simple_animal/hostile/retaliate/Triangle.dm

```diff
@@ -0,0 +1,30 @@
+/mob/living/simple_animal/hostile/retaliate/Triangle
+	name = "Triangle spider"
```

Comment:
The name variable needs to be lower case either, otherwise the game will display it like "Triangle spider hits you!" instead of "The triangle spider hits you!"